CODEOWNERS

```diff
@@ -1,9 +1,8 @@
 # Lines starting with '#' are comments.
 # Each line is a file pattern followed by one or more owners.
 
-# These owners will be the default owners for everything in the repo.
-*       @jamesgeddes
-*       @CarwynNelson
+# These owners will be the default owners for everything in the repo.       
+*       @GeekZoneHQ/q
```

Comment:
q as in

![q from star trek](https://vignette.wikia.nocookie.net/dbxfanon/images/2/2a/Q_(Star_Trek).jpeg/revision/latest?cb=20160417235618)

Reply:
Yep!